tests/integration/validatorTests.py

```diff
@@ -39,55 +40,48 @@ def setUpClass(cls):
         column1.required = True
         column1.field_type = stringType
         column1.file_id = 1
-        cls.interfaces.validationDb.populateFile(column1)
```

Comment:
It's not happening elsewhere as these columns are built manually (not pulled from DB).  It's possible that validate no longer uses the file type relationship, or this may result in these rules being skipped because the file type doesn't match.